src/components/Exercise.tsx

```diff
@@ -86,41 +95,49 @@ export const Exercise = ({
     }
   }, [scrollToQuestion, exercise]);
 
-  return (<StyledTaskStepCard
+  React.useEffect(() => {
+    typesetExercise();
+  }, [exercise]);
```

Comment:
you could not have this here and just call the context from `Content`

Reply:
Yeeaaaa I like that a lot better!